### PR TITLE
Improve pip install

### PR DESCRIPTION
**Pull request by patricksnape**
Allow pip installing *without* requiring Cython (which will now work from the sdist upload). Stole the requisite function from the Cython documentation.

### CODE REVIEW

**Comment by patricksnape** [comment]
retest this please

**Comment by patricksnape** [comment]
@jabooth look good?

File: menpo/feature/windowiterator.pyx [hunk 1]
```diff
@@ -1,5 +1,4 @@
-# distutils: language = c++
-# distutils: sources = menpo/feature/cpp/ImageWindowIterator.cpp menpo/feature/cpp/WindowFeature.cpp menpo/feature/cpp/HOG.cpp menpo/feature/cpp/LBP.cpp
+# distutils: sources =
```
Comment (jabooth) [inline menpo/feature/windowiterator.pyx]: intentionally left empty?

File: setup.py [hunk 2]
```diff
@@ -55,9 +96,7 @@
       install_requires=install_requires,
       package_data={'menpo': ['data/*',
                               'feature/cpp/*.cpp',
-                              'feature/cpp/*.h',
-                              'transform/piecewiseaffine/fastpwa/*.c',
-                              'transform/piecewiseaffine/fastpwa/*.h'],
-                    '': ['*.pxd', '*.pyx']},
+                              'transform/piecewiseaffine/fastpwa/*.cpp'],
+                    '': ['*.pxd', '*.pyx', '*.h']},
```
Comment (jabooth) [inline setup.py]: can't we simplify this down now?
```diff
       package_data={'menpo': ['data/*',
-                              'feature/cpp/*.cpp',
-                              'transform/piecewiseaffine/fastpwa/*.cpp'],
-                    '': ['*.pxd', '*.pyx', '*.h']},
+                    '': ['*.pxd', '*.pyx', '*.cpp', '*.h']},
```
(i.e. we _always_ want `.cpp` files, just like `.pyx` etc)

Comment (patricksnape) [inline setup.py]: Good catch.

**Comment by patricksnape** [comment]
Fixed - thanks @jabooth

**Pull request merged**